packages/@o3r/schematics/schematics/ng-add/index.ts

```diff
@@ -16,15 +15,15 @@ import { NodePackageName } from '@angular-devkit/schematics/tasks/package-manage
 class DevInstall extends NodePackageInstallTask {
   /** @inheritDoc */
   public toConfiguration() {
-    const installOptions = getPackageManager();
+    const installOptions = process.env && process.env.npm_execpath && process.env.npm_execpath.indexOf('yarn') === -1 ? 'npm' : 'yarn';
```

Comment:
why not importing it from the `src`?

Reply:
could do now that it is in the schematics

Reply:
Nope, you can't refer to the source until the peerdep are resolved, hence I cannot use the one from src.